card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -605,16 +609,32 @@ internal class DefaultCardDelegate(
 
     private fun makeCvcUIState(cvcPolicy: Brand.FieldPolicy?): InputFieldUIState {
         Logger.d(TAG, "makeCvcUIState: $cvcPolicy")
-        return when {
-            isCvcHidden() -> InputFieldUIState.HIDDEN
-            cvcPolicy?.isRequired() == false -> InputFieldUIState.OPTIONAL
-            else -> InputFieldUIState.REQUIRED
+
+        return when (componentParams.cvcVisibility) {
```

Comment:
Let's make sure we use the default behavior with locally detected cards and only change the cvc visibility when it's from bin lookup

Reply:
done